cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,18 @@
 /// <reference types="cypress" />
-
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
-
-  });
+  beforeEach(() => {
+ cy.visit('https://react-redux.realworld.io')
+});
+it('should provide an ability to log in', () => {
+cy.contains('a', 'Sign in').click();
+cy.url().should('include','/login');
+cy.get(':nth-child(1) > .form-control')
+ .type('userNam@gmail.com');
```

Comment:
Email is hard coded. Try to replace this with environment variables or config files to prevent exposing sensitive data.